units/XNL0001/XNL0001_script.lua

```diff
@@ -403,6 +414,10 @@ XNL0001 = Class(ACUUnit) {
         end
     end,
 
+    ---@param self XNL0001
+    ---@param location number
+    ---@param probeType string
+    ---@param data any
```

Comment:
As I understand from the LaunchProbe function and what is used in this function:
```suggestion
    ---@param data { Lifetime: number, CoolDownTime: number }
```